examples/chip-tool/config/PersistentStorage.cpp

```diff
@@ -239,3 +240,29 @@ CHIP_ERROR PersistentStorage::SetLocalNodeId(NodeId value)
     uint64_t nodeId = Encoding::LittleEndian::HostSwap64(value);
     return SyncSetKeyValue(kLocalNodeIdKey, &nodeId, sizeof(nodeId));
 }
+
+CATValues PersistentStorage::GetLocalCATs()
```

Comment:
What are local CATs?  CATs will appear in certificates and in ACLs.

I don't see how this fits into that.

Reply:
I renamed this from  `GetLocalCATs()` to `GetCommissionerCATs()`. This refers to the CATs that will be included in the Commissioner OpCert.